### ISSUE TITLE
Iframe freeze, can i detect it?

### ISSUE THREAD

**Issue by Jumbo125** [AlexxIT/go2rtc#574]
Hy 
i use rtc to embed my cam via iframe.
the problem:
sometimes my video freeze. i see the timestamp from the camera in the left corner. so i see, when the video frezze.
i write a script for a button -> when i press the button, the video src will be reload. The video works.

summary:
the video embed frezze. when i reload the embed, it works great!

So my question:
can i detect when the video frezze, to reload it automatically?
i don't want to press refresh every 20minutes. The big problem: i don't knoow when it freeze.... i write a function to reload all 10 minutes. thats help, but it'S not good....

**Comment by AlexxIT** [comment]
What streaming technology can you see in top right corner?

**Comment by Jumbo125** [comment]
RTC

**Comment by AlexxIT** [comment]
I don't think it related with iframe. It may be problem of hi camera bitrate.

**Comment by Jumbo125** [comment]
hmmm.... if i press realod, it works great.... but i can't always sit in front of the tablet and check if it freeze, to reload the iframe src

**Comment by AlexxIT** [comment]
It's know problem when RTC can freeze on hi bitrates. Because UDP can't handle big traffic.

**Comment by Jumbo125** [comment]
Okay, i see

Can I reduce the Bitrate by stream url or do i need to change the Bitrate in the camera settings?

**Comment by AlexxIT** [comment]
Better to change the camera settings

**Comment by Jumbo125** [comment]
puh... i need to have a look, if it is possible

**Comment by Jumbo125** [comment]
hello alexxIT

thank you for your helo

i tried a lot of things. the camera can't change the bitrate.
so i need to find a other solution.
can you tell me, where the stream.html file is?
i can html, js (jquery) and php
maybe i can create a solution, but i need to do it in this file. When i embed the iframe i can not edit any options with js, because the origin privacy error. so i need to set my ideas in the stream.html file

**Comment by AlexxIT** [comment]
https://github.com/AlexxIT/go2rtc/tree/master/www

**Comment by Jumbo125** [comment]
> https://github.com/AlexxIT/go2rtc/tree/master/www

that i had found by myself. but i can't found the driectory on my linux pc....
where is it save, when i start the release file from you?

**Comment by Jumbo125** [comment]
i have a idea, why i can'T find it on my linux system

The go2rtc software use the stream.html file from your github server.....
is it possbile to run it onyl local?
to download all files and use all files from my pc

**Comment by AlexxIT** [comment]
Set `static_dir` and copy content of www folder there
https://github.com/AlexxIT/go2rtc#module-api

**Comment by Jumbo125** [comment]
wow! you are great. This project is so big! unbelievable. Your knowhow is really great! 
are the api setting lice static_dir also in the go2rtc.yaml file?
before i config the streams?

do you work in your job as software engineer?
